File: 02_ProgrammingFundamentals/_02_DataTypesAndVariables/_01_Lecture/_11_RefactorVolumeOfPyramid.java
package _02_ProgrammingFundamentals._02_DataTypesAndVariables._01_Lecture;

import java.util.Scanner;

public class _11_RefactorVolumeOfPyramid {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input values
        double length = Double.parseDouble(scan.nextLine()), width = Double.parseDouble(scan.nextLine()), height = Double.parseDouble(scan.nextLine());

        // Print result
        System.out.print("Length: ");
        System.out.print("Width: ");
        System.out.print("Height: ");
        System.out.printf("Pyramid Volume: %.2f", (length * width * height) / 3);

        scan.close();
    }
}

File: 24_C#Fundamentals/_02_DataTypesAndVariables/_01_Lecture/_11_RefactorVolumeOfPyramid/_11_RefactorVolumeOfPyramid.cs
using System;

public class _11_RefactorVolumeOfPyramid
{
    public static void Main(string[] args)
    {
        // Read input values
        double length = double.Parse(Console.ReadLine()), width = double.Parse(Console.ReadLine()), height = double.Parse(Console.ReadLine());

        // Print result
        Console.Write("Length: ");
        Console.Write("Width: ");
        Console.Write("Height: ");
        Console.WriteLine($"Pyramid Volume: {(length * width * height) / 3:F2}");
    }
}
